props/saber_sabersense_buttons.h

```diff
@@ -0,0 +1,1017 @@
+							click AUX while in VOLUME MENU.
+							Volume adjusts in increments per click.
+							You must exit VOLUME MENU to resume using saber normally.
+							
+FUNCTIONS WITH BLADE ON
+Blade lockup			Press and hold AUX.
+Blade tip drag			Press and hold AUX while blade is pointing down.
+Force Effect (Quote)	Fast double-click MAIN, hilt pointing down.
+							Plays random Force effect (usually character quote).
+Character Quote			Fast double-click MAIN, hilt pointing up.
+							Plays sequential character quote.
+Lightning block			Double-click MAIN and hold.
+Melt         			Hold MAIN and push blade tip against wall (stab).
+Blaster blocks			Short click AUX.
+Enter multi-blast mode	Hold MAIN while swinging for one second and release. 
+							Saber will play two quick blasts confirming mode.
+							Swing blade to trigger blaster block.
+							To exit multi-blast mode, fast single click AUX.
+
+COLOUR CHANGE FUNCTIONS WITH BLADE ON
+Enter/Exit COLOUR MENU	Hold MAIN then quickly click AUX and release both
+						buttons simultaneously. Or fast triple-click MAIN.
+							Announcement confirms you are in the COLOUR MENU.
+Cycle to next colour    Rotate hilt whilst in COLOUR MENU until desired colour is reached.
+							Most presets have 12 colour options.
+Alt Exit COLOUR MENU	Hold MAIN and wait. 
+							Announcement confirms you are exiting COLOUR MENU.
+							You must exit COLOUR MENU to resume using lightsaber normally.
+
+							
+===========================================================		
+========================= DEFINES =========================
+#define SABERSENSE_FAKE_BLADE_ID
+Replaces regular BladeID and allows toggling between two different 
+blade/preset arrays regardless of actual BladeID status.
+
+#define SABERSENSE_NO_LOCKUP_HOLD	
+Applicable to two-button mode only, reverts to lockup being 
+triggered by clash while holding aux.
+
+GESTURE CONTROLS
+There are four gesture types: Twist, Stab, Swing and Thrust.
+Gesture controls bypass all preon effects.
+Below are the options to add to the config to enable the various gestures. MM.
+#define SABERSENSE_TWIST_ON
+#define SABERSENSE_TWIST_OFF
+#define SABERSENSE_STAB_ON
+#define SABERSENSE_SWING_ON
+#define SABERSENSE_THRUST_ON
+
+Tightened click timings
+I've shortened the timeout for short and double click detection from 500ms
+to 300ms.  I think it feels more responsive this way but still gives enough
+timeout to ensure all button actions can be achieved consistently
+I've included all button timings so they can be easily tweaked to suit
+individual tastes. MM.
+
+============================================================
+============================================================
+*/
+#ifndef PROPS_SABER_SABERSENSE_BUTTONS_H
+#define PROPS_SABER_SABERSENSE_BUTTONS_H
+
+
+//  Used instead of BladeID and allows toggle switching 
+//  between two blade/preset arrays manually irrespective
+//  of actual BladeID status.
+	#ifdef SABERSENSE_FAKE_BLADE_ID
```

Comment:
207 to 220 are super-weirdly indented, please fix.

Reply:
Hopefully now sorted.